core/prisma/migrations/20241105184158_copy_membership_data/migration.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+INSERT INTO "community_memberships"
+    SELECT "id", "role", "communityId", "userId", "createdAt", "updatedAt" FROM "members"
+    ON CONFLICT ("id") DO NOTHING;
```

Comment:
the conflicts you are anticipating here are community_memberships that would have been created in tandem with "legacy" memberships due to the previous pr, correct?

Reply:
exactly. not a super likely scenario but it could happen!